lib/appcues_flutter.dart

```diff
@@ -346,8 +358,13 @@ class Appcues {
           // into global coordinates for the screen.
           Rect transformToRoot(Rect rect, SemanticsNode? node) {
             var transform = node?.transform;
+            var parent = node?.parent;
             if (transform == null) {
-              return rect;
+              if (parent != null) {
+                return transformToRoot(rect, parent);
+              } else {
+                return rect;
```

Comment:
this fixes the issue with the 1st tab not showing in the correct spot. The `node.transform` was `null` now for that tab (not sure why) and i was causing the untransformed `rect` to be returned, thus it was at (0,0).

Now, we are more robust, and continue to recurse as long as the `parent` is not null - getting the properly transformed global coordinates for the view.